blocks/api/raw-handling/index.js

```diff
@@ -137,6 +141,7 @@ export default function rawHandler( { HTML, plainText = '', mode = 'AUTO', tagNa
 		] ) );
 
 		piece = deepFilterHTML( piece, [
+			singleItemListConverter,
```

Comment:
I think the "filter" was implemented in separate so the changes are isolated, but as we already have a listReducer  filter would it make sense for singleItemListConverter to be part of the other filter?

Not related to this changes but we have 3 consecutive calls to deepFilterHTML, Should we just join them all? One has a comment specifying it should be the first but even if we join them all we can still have comment saying msListConverter should be the first filter.

Reply:
I don't mind moving it out of that particular `deepFilterHTML` call, but differences in the document mean that the `ms-word-online` integration test fails if the filter is placed in the same `deepFilterHTML` call as `listReducer`, …, `inlineContentConverter`.

That said, I agree there's room to improve the overall efficiency in that bit of `rawHandler`.